flask_ninja/models.py

```diff
@@ -152,7 +152,7 @@ class MediaType(BaseModel):
 class ParameterBase(BaseModel):
     description: Optional[str] = None
     required: Optional[bool] = None
-    deprecated: Optional[bool] = None
+    deprecated: Union[Any, str, bool, None] = None
```

Comment:
`depreacted` was added to `pydantic.fields.FieldInfo` in `pydantic` somewhere around version `2.7.0`